src/Elmah.AzureTableStorage/Elmah.AzureTableStorage.csproj

```diff
@@ -31,32 +31,39 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Elmah">
+    <Reference Include="Elmah, Version=1.2.14706.0, Culture=neutral, processorArchitecture=MSIL">

```

Comment:
Can you make sure all version numbers are the same or greater? Unless there is a valid reason for using a lower version number?